docs/guide/localization/static.md

```diff
@@ -0,0 +1,199 @@
+
+If the only localization key we registered in our project was `relatedArticles` from above, our files would look like:
+
+<AposCodeBlock>
+  ```json
+  {
+    "relatedArticles": "Related articles"
+  }
+  ```
+  <template v-slot:caption>
+    /modules/localization/i18n/en.json
+  </template>
+</AposCodeBlock>
+
+<AposCodeBlock>
+  ```json
+  {
+    "relatedArticles": "Artículos relacionados"
+  }
+  ```
+  <template v-slot:caption>
+    /modules/localization/i18n/es.json
+  </template>
+</AposCodeBlock>
+
+When rendering the show page template, Apostrophe will look in these files, find the registered key and replace it with the correct string based on the active locale.
+
+## Using namespaces
+
+A l10n **namespace** is a prefix on localization keys that makes it harder to accidentally override. In project-level l10n namespacing is not really necessary since there are not additional layers of work that might override translation there.
+
+Namespacing can be useful if you are building your own modules with hard-coded strings that you intend to publish. When that module is installed in a project later it would be less likely that the project will change them accidentally.
+
+To use a l10n namespace, include it in the `i18n` option object in the module where your JSON files are with the `ns` setting.
+
+<AposCodeBlock>
+  ```javascript
+  module.exports = {
+    options: {
+      i18n: {
+        ns: 'myTeam'
+      }
+    }
+  };
+  ```
+  <template v-slot:caption>
+    /modules/localization/index.js
+  </template>
+</AposCodeBlock>
+
+**You do not need to include this namespace in the l10n JSON files.** If the JSON files are in the module directory where this setting appears, Apostrophe will know that the keys in those files should be namespaced.
+
+Then when you use the localization keys in template files (or elsewhere), start each key with the namespace:
+
+```django
+<h2>__t('myTeam:relatedArticles')</h2>
+```
+
+Apostrophe will then treat keys with the namespace differently from the same key without the namespace (`myTeam:relatedArticles` vs. `relatedArticles`). If someone uses the version *without* the namespace it will not overwrite the version *with* the namespace.
+
+## Localizing the Apostrophe user interface
+
+The Apostrophe user interface contains many registered strings, currently localized to English. We will be working to provide more localization files, but if you are interested in adding l10n files to your project for the UI, you are welcome to do that.
+
+Add JSON files for the locales as normal in the project-level `modules/@apostrophecms/i18n` module directory. It's important to use this directory since it already has the `'apostrophe'` namespace configured. The UI keys all use this namespace. In each JSON file, copy the contents of [the Apostrophe core l10n file](https://github.com/apostrophecms/apostrophe/blob/main/modules/@apostrophecms/i18n/i18n/en.json) to get all the keys. You can then start translating each string.
```

Comment:
You're welcome to use this directory, because it already has the namespace configured, but I wouldn't say it's important. In fact, if you're thinking of sharing your translations in npm for others to use, you'll be happiest starting off with your own module and setting `ns: 'apostrophe'` there, which multiple modules are welcome to do.